src/modifiers/Modifier.js

```diff
@@ -50,8 +50,34 @@ class Modifier {
   get maxIterations() {
     return 1000;
   }
+
+  /**
+   * No default values present
+   *
+   * @param {StandardDice|RollGroup} _context The object that the modifier is attached to
+   *
+   * @returns {object}
+   */
+  defaults(_context) {
+    return {};
+  }
   /* eslint-enable class-methods-use-this */
 
+  /**
+   * Processing default values definitions
+   *
+   * @param {StandardDice|RollGroup} _context The object that the modifier is attached to
+   *
+   * @returns {void}
+   */
+  ensureParameters(_context) {
```

Comment:
I'm wondering if this would be better named something like `setDefaults`?
I'm not brilliant at naming things, so I'm not saying you have to change it, just a point of discussion.

Reply:
I'm sure that there should be a better name, than `ensureParameters` (I'm not proud about both parts of that name :) ), but the reason I didn't go with `setDefaults` is that it doesn't actually set them if the user-passed value is already set. So calling it `set` would be a misnomer.